app/models/user.py

```diff
@@ -451,6 +453,11 @@ def rocket_chat_username(self):
     def rocket_chat_password(self):
         return get_serializer().dumps(f'rocket_chat_user_{self.id}')
 
+    @property
+    def has_group_role(self):
+        group = UsersGroupsRoles.query.filter_by(user_id=self.id)
+        return group is not None
```

Comment:
In fact, this gives a user organizer access to all events because it doesn't even check for a particular event.

In fact it will always return True because the query doesn't even execute, it is always not None